pkg/datapath/loader/netlink.go

```diff
@@ -117,8 +118,7 @@ func graftDatapath(ctx context.Context, mapPath, objPath, progSec string) error
 	}()
 
 	// FIXME: replace exec with native call
-	// FIXME: only key 0 right now, could be made more flexible
-	args := []string{"exec", "bpf", "graft", mapPath, "key", "0",
+	args := []string{"exec", "bpf", "graft", mapPath, "key", strconv.FormatInt(int64(key), 10),
```

Comment:
is there a reason why key can't be passed as a `string` instead of an `int`?

Reply:
'Key' represents the  key of BPF_MAP_TYPE_PROG_ARRAY map. I think it should be int in golang.